pkg/ipmi/set_session_privilege_level.go

```diff
@@ -0,0 +1,84 @@
+package ipmi
+
+import (
+	"fmt"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+)
+
+// SetSessionPrivilegeLevelReq implements the Set Session Privilege Level command, specified in section
+// 18.16 of v1.5 and 22.18 of v2.0.
+type SetSessionPrivilegeLevelReq struct {
+	layers.BaseLayer
+
+	// PrivilegeLevel indicates the privilege level to switch to.
+	// Set to 0 to retrieve the present privilege level.
```

Comment:
Given the default value is 0, let's avoid mentioning that. It's better to hide the underlying value and refer to the `PrivilegeLevel*` constants. How about rephrasing this sentence to "Omitting this field will retrieve the current privilege level without modification. PrivilegeLevelHighest and PrivilegeLevelCallback are invalid values."